Base/usr/share/man/man7/KeyboardShortcuts.md

```diff
@@ -0,0 +1,65 @@
+![Keyboard Icon](/res/icons/32x32/app-keyboard-settings.png)
+
+# Keyboard Shortcuts
+
+This is a cheatsheet of common keyboard shortcuts in SerenityOS
+
+Related: [Tips and Tricks](help://man/7/Tips-and-Tricks)
+
+### Keywords
+* `Super` means the ⊞ Windows or ⌘ Command key
```

Comment:
Hmmm, is "Super" or "Meta" the more common term? I see "Meta" all over Linux desktop documentation, for instance.

Reply:
[Andreas uses "Super"](https://www.youtube.com/watch?v=mco45wl0KA4&lc=UgyJ6Rs8LOMU8smMtT54AaABAg.9l_sy2K9iNR9la3XFqPugK), but I will include "Meta" keyword meaning